skimage/io/collection.py

```diff
@@ -37,13 +38,18 @@ def concatenate_images(ic):
     ------
     ValueError
         If images in `ic` don't have identical shapes.
+
+    Notes
+    -----
+    ``concatenate_images`` receives any iterable object containing images,
+    including ImageCollection and MultiImage, and returns a NumPy array.
     """
-    all_images = [img[np.newaxis, ...] for img in ic]
+    all_images = [image[np.newaxis, ...] for image in ic]
     try:
-        ar = np.concatenate(all_images)
+        array_cat = np.concatenate(all_images)
```

Comment:
why is stack not used?

```python
In [7]: np.stack([a, b])                                                                                 
Out[7]: 
array([[[0., 0., 0.],
        [0., 0., 0.],
        [0., 0., 0.]],

       [[1., 1., 1.],
        [1., 1., 1.],
        [1., 1., 1.]]])

In [8]: np.stack([a, b]).shape                                                                           
Out[8]: (2, 3, 3)
```

Reply:
This really was only refactoring. I didn't think about it.

Reply:
stack did not exist when this code was written.